libs/ui-lib/src/lib/custom-file-upload/custom-file-upload.scss

```diff
@@ -0,0 +1,50 @@
+@import 'variables';
+
+.imageURL {
+  margin: 1.8em 5em;
+  box-sizing: border-box;
+  width: auto;
+}
+.imageURL-active {
+  width: '100%';
+  display: 'block';
+  border-radius: '6.5px';
+}
+
+.custom-file-upload {
+  display: block;
+  box-sizing: border-box;
+  cursor: pointer;
+  border-radius: 8px;
+  border: 1.5px $primary-color solid;
```

Comment:
Partial pixel amounts sometimes leave whitespace. If you do this suggestion:
```suggestion
  border: 1px $primary-color solid;
```

And set:
```scss
.custom-file-upload-image {
  ...
  border-radius: 7px; // instead of 6.5px
}
```
You get this:
![image](https://user-images.githubusercontent.com/23544999/99766007-425db880-2ace-11eb-846c-eb5da2c28645.png)

Instead of this:
![image](https://user-images.githubusercontent.com/23544999/99765962-2bb76180-2ace-11eb-9294-f3292068991e.png)

Reply:
Actually I noticed in the mockup there is not border when the image is uploaded. It also looks cleaner without it, so I remove the border when an image has been uploaded.

Reply:
alright sounds good